search_test.go

```diff
@@ -453,6 +446,18 @@ func TestNestedBooleanSearchers(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	defer func() {
+		err = idx.Close()
```

Comment:
idx will be nil if test failed at 440..

Reply:
If the test fails at line 440, this defer won't be invoked :)